index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tic Tac Toe</title>
+<link rel="stylesheet" type="text/css" href="style.css"></head>
+<body>
+<h1>Tic Tac Toe</h1>
+<div id="description"> <p>By Audrey Andoy</p>
```

Comment:
I would do the same thing here, just so that its easier to keep track of which elements are inside which divs:

```
    <div id="description">
        <p>By Audrey Andoy</p>
        <p>Click a space below to begin playing! Have fun :) </p>
    </div>
```